numpy/core/multiarray.py

```diff
@@ -98,6 +98,10 @@ def empty_like(prototype, dtype=None, order=None, subok=None):
         type of 'a', otherwise it will be a base-class array. Defaults
         to True.
 
+        .. versionadded:: 1.17.0
+    shape : int or sequence of ints, optional.
+        Overrides the shape of the result.
+
```

Comment:
The versionadded goes below the attribute, [here](https://6247-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.empty_like.html?highlight=empty_like) is how this is rendered. The *New in version 1.17.0.* is showing in the previous parameter

Reply:
I see it now, the format of linespaces is a bit misleading there. I'll fix it now.